createdisk-library.sh

```diff
@@ -161,7 +161,7 @@ function install_additional_packages() {
         # Download the hyperV daemons dependency on host
         local pkgDir=$(mktemp -d tmp-rpmXXX)
         mkdir -p ${pkgDir}/packages
-        sudo yum download --downloadonly --downloaddir ${pkgDir}/packages "$*" --resolve
+        sudo yum download --downloadonly --downloaddir ${pkgDir}/packages $* --resolve
```

Comment:
The package names should not contain spaces, so yes, the double quotes do not seem needed